### PR TITLE
Allow the observer to be deployed independently from Osiris

### PR DESCRIPTION
**Pull request by CermakM**
Observer can now pass kubernetes event to Osiris directly

- get rid of Osiris dependency

### CODE REVIEW

**Comment by CermakM** [comment]
Addresses #3

**Comment by goern** [comment]
recheck

File: app.py [hunk 1]
```diff
@@ -43,78 +41,36 @@
...
-_REQUESTS_MAX_RETRIES = 10
-
-
-# kubernetes configuration
-
-_SERVICE_TOKEN_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/token"
-_SERVICE_CERT_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-
-if Path(_SERVICE_TOKEN_FILENAME).exists():  # in-cluster configuration
+    _NAMESPACE = os.getenv("MIDDLETIER_NAMESPACE", None)
```
Comment (goern) [inline app.py:73]: is this indentation level correct?

Comment (CermakM) [reply app.py:73]: Think it was, but it is no longer applicable anyway, in the next commit I am gonna get rid of all that.

**Review by goern — APPROVED** [review]

**Pull request merged**